config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -111,6 +111,11 @@ function create_libvirt_image_from_scratch() {
     PODVM_IMAGE_PATH=/payload/podvm-libvirt.qcow2
     cp -pr "${CAA_SRC_DIR}"/podvm/output/*.qcow2 "${PODVM_IMAGE_PATH}"
 
+    # Get RVPS values from qcow2
+    if [ "$SE_BOOT" == "true" ]; then
```

Comment:
do we need to explicitly check for arch as well here? if the admin accidentally enables `SE_BOOT` for non-s390x libvirt, maybe this case might be applicable? but I think it's okay to not have it as well..

Reply:
I had added it to cater (https://github.com/openshift/sandboxed-containers-operator/pull/444#discussion_r1740372417). I will check for some alternative.

Reply:
I have added check for s390x also which should cater it.